tests/rule_based_profiler/data_assistant/test_data_assistant_runner.py

```diff
@@ -105,3 +105,36 @@ def test_onboarding_data_assistant_runner_top_level_kwargs_explicit_none(
     }
 
     assert "user_id" not in categorical_columns_used
+
+
+@pytest.mark.big
+def test_onboarding_data_assistant_runner_using_validator(
```

Comment:
This test really exemplifies how simple it gets with the `validator` vs `batch_request`. Thank you very much for this